app/lib/actions/katello/host/auto_attach_subscriptions.rb

```diff
@@ -10,6 +10,9 @@ def queue
 
         def plan(host)
           action_subject(host)
+          if ::Organization.find_by_id(host.organization_id).simple_content_access?
+            fail ::Katello::HttpErrors::BadRequest, _("Auto attach is disabled in Simple Content Access mode")
```

Comment:
We should be clear that it's the organization in simple content access mode: `This host's organization is in Simple Content Access mode. Auto-attach is disabled` is a little more precise. Let me know what you think